src/kudu/consensus/consensus_meta_manager-test.cc

```diff
@@ -115,7 +115,7 @@ TEST_F(ConsensusMetadataManagerTest, TestLoadOrCreate) {
   // LoadOrCreateCMeta() call, above, not the second call.
   ASSERT_EQ(kInitialTerm, cmeta->current_term());
   ASSERT_TRUE(MessageDifferencer::Equals(config_, cmeta->CommittedConfig()))
-      << DiffRaftConfigs(config_, cmeta->CommittedConfig());
+      << DiffRaftConfigs(config_, cmeta->CommittedConfig(), nullptr);
```

Comment:
something triggered this change?

Reply:
Yep, had to change the API to get the evicted peers